packages/odie-client/src/components/message/sources.tsx

```diff
@@ -12,6 +12,18 @@ export const Sources = ( { message }: { message: Message } ) => {
 	const sources = useMemo( () => {
 		const messageLength = message?.context?.sources?.length ?? 0;
 		if ( messageLength > 0 ) {
+			// Record TrainTracks render events
+			let uiPosition = 0;
+			message.context?.sources?.forEach( ( source: Source ) => {
```

Comment:
Second parameter of forEach is the index, you don't need to track the uiPosition :)